tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -241,7 +246,11 @@ internal static bool CheckRegistry(IRegistryService? registry = null)
 
                 // Check that the profiler is properly registered
                 ok &= CheckClsid(registry, ClsidKey);
-                ok &= CheckClsid(registry, Clsid32Key);
+
+                if (tracerVersion == null || tracerVersion < new Version("2.14.0.0"))
+                {
+                    ok &= CheckClsid(registry, Clsid32Key);
```

Comment:
This only checks for the 32-bit value. I think we also need to skip the 64-bit check in line 248 above:
```csharp
ok &= CheckClsid(registry, ClsidKey);
```